src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -96,23 +132,31 @@ public LocalDateTime getDateInSpecificTimeZone(String dateInString, String zone)
      * OffsetDateTime советуют использовать при записи даты в базу данных.
      */
     public OffsetDateTime offsetDateTime(LocalDateTime localTime) {
-        return OffsetDateTime.now();
+        return OffsetDateTime.of(localTime, ZoneOffset.of("+02:00"));
     }
 
     /**
      * Дана строка в виде "yyyymmdd".
      * Необходимо вернуть Optional даты в LocalDate формате
      */
-    public Optional<LocalDate> parseDate(String date) {
-        return Optional.empty();
+    public Optional<LocalDate> parseDate(String date) throws DateTimeParseException {
+        try {
+            return Optional.of(LocalDate.parse(date, FORMATTER1));
```

Comment:
```suggestion
            return Optional.of(LocalDate.parse(date, DateTimeFormatter.BASIC_ISO_DATE));
```
you can use existing constant here